Assets/UI/Scripts/Options.gd

```diff
@@ -0,0 +1,23 @@
+extends Control
+
+var parent = null
```

Comment:
This name is misleading, since it's not actually the parent. Maybe `previous_menu`?

Reply:
Since it is the same name for e.g. NewGameUI.gd, I'll leave it as-is.